modules/lang-painless/src/main/antlr/PainlessParser.g4

```diff
@@ -156,11 +156,11 @@ postdot
     ;
 
 callinvoke
-    : DOT DOTID arguments
+    : nullSafe=COND? DOT DOTID arguments
```

Comment:
I thought about doing this in the lexer instead but went this way because I figured they were about equal. If you prefer lexer I'm certainly fine moving it.